scripts/deploy_controller.sh

```diff
@@ -11,5 +11,8 @@ export SERVICE_BASE_URL=${SERVICE_BASE_URL:-"http://${SERVICE_URL}:${SERVICE_POR
 mkdir -p build
 
 if [ "${DEPLOY_TARGET}" == "ocp" ]; then
-    skipper run "scripts/ocp.sh deploy_controller $OCP_KUBECONFIG $SERVICE_BASE_URL $NAMESPACE $CONTROLLER_OCP"
+    CLUSTER_ID=$(curl -s $SERVICE_BASE_URL/api/assisted-install/v1/clusters | jq -r '.[0] | .id')
```

Comment:
I thought we are gonna do it at the beginning of the main function in a loop. Otherwise, it is racy: assisted-service on OCP might not have created the cluster yet when you are quiring it for cluster id

Reply:
I'm not sure it's an actual risk, as the query is called after the service and ui are ready. So it should be plenty of time for the cluster to be created in db.